example_school_library_decorator/main.rb

```diff
@@ -42,17 +30,6 @@ def list_all_rentals
     end
   end
 
-  def create_book
-    print "Book title: "
-    title = gets.chomp
-
-    print "Book author: "
-    author = gets.chomp
-
-    @books << Book.new(title, author)
-    puts "Book created succesfully"
-  end
-
   def create_rental
     if @books.empty?
```

Comment:
- [ ] I noticed that when I run your application, it breaks when a user tries to create a rental as shown in the screen, this is because you have excluded the @books and @people instance variables that should be present in your Bookstore || App class. Kindly fix this by creating these instance variables so that your application does not break
![Screenshot 2021-10-20 at 14 07 24](https://user-images.githubusercontent.com/57832450/138094801-846857ee-9623-4a41-9a17-7cb196ccc6b9.png)
.